addons/mail/static/src/owl/components/composer/composer.js

```diff
@@ -341,6 +350,17 @@ class Composer extends Component {
         this._postMessage();
     }
 
+    _setTextInputHtmlContent(textInputContent) {
+        //Removing unwanted extra spaces from message
+        let value = _.escape(textInputContent).trim();
+        value = value.replace(/(\r|\n){2,}/g, '<br/><br/>');
```

Comment:
Does that work on windows? I think there both `\r` and `\n` might be present at the same line and should only lead to one new line.

Reply:
We'll see that later